docs/marketplace/dev/howtos/how-to-add-new-column-type.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
please rename the filenames so it is easier to navigate them:
how-to-add-new-column-type => how-to-add-new-guitable-column-type
how-to-add-new-filter-type => how-to-add-new-guitable-filter-type
etc.